e2e/framework/manager/rook_test_infra.go

```diff
@@ -170,30 +170,30 @@ func (r *rookTestInfraManager) copyImageToNode(containerId string, imageName str
 		cmdOut := utils.ExecuteCommand(objects.CommandArgs{Command: "docker", CmdArgs: []string{"pull", imageName}})
 
 		if cmdOut.Err != nil || cmdOut.ExitCode != 0 {
-			return fmt.Errorf("Failed to pull image")
+			panic(fmt.Errorf("Failed to pull image"))
```

Comment:
consider including `imageName` in the panic output (this applies to other panics below too)